nvflare/private/fed/client/client_engine.py

```diff
@@ -48,27 +48,26 @@ def _remove_custom_path():
 class ClientEngine(ClientEngineInternalSpec):
     """ClientEngine runs in the client parent process."""
 
-    def __init__(self, client, client_name, args, rank, workers=5):
+    def __init__(self, client, args, rank, workers=5):
```

Comment:
if we could, it might be clearer to add a typehint so users know "client" should be a "class FederatedClientBase"

Reply:
done